data_dictionary/data_dictionary.hh

```diff
@@ -55,11 +55,13 @@ class impl;
 class user_types_metadata;
 class keyspace_metadata;
 
+// Should not be returned directly to the user
 class no_such_keyspace : public std::runtime_error {
 public:
     no_such_keyspace(std::string_view ks_name);
 };
 
+// Should not be returned directly to the user
```

Comment:
I don't see this being caught anywhere - I suspect it may be returned directly so this comment doesn't apply